prefs.js

```diff
@@ -123,6 +108,26 @@ function buildPrefsWidget() {
     layout.attach(previewLabel, 0, row, 1, 1);
     layout.attach(previewInput, 1, row++, 1, 1);
 
+    // Double width previews
+    let doubleWidthLabel = new Gtk.Label({
+        label: _("     Use double width previews on sides in 4 column mode"),
```

Comment:
Horrible hack, but I am not familiar enough with GTK and honestly didn't want to spend hours on it :) Feel free to change this to do the spacing properly.